lib/v2/screens/profile_screens/edit_name/interactor/usecases/update_profile_use_case.dart

```diff
@@ -9,12 +9,12 @@ class UpdateProfileUseCase {
   Future<Result> run({required String? newName, required ProfileModel profile}) {
     return _profileRepository.updateProfile(
       accountName: settingsStorage.accountName,
-      nickname: newName,
-      image: profile.image,
-      story: profile.story,
-      roles: profile.roles,
-      skills: profile.skills,
-      interests: profile.interests,
+      nickname: newName ?? '',
```

Comment:
why is this needed?

Reply:
update profile transaction can't send null for empty fields.

> Apparently after the migration to null safety put the parameters of the updateProfile call as nullables and not required, this caused null values ​​to be sent in the transaction instead of empty strings.

Reply:
I see. 
Bad services.